docs/050-breaking-changes.rst

```diff
@@ -0,0 +1,31 @@
+********************************
+Solidity v0.5.0 Breaking Changes
+********************************
+
+Solidity version 0.5.0 introduces many breaking changes.
+This section shows how to update your code regarding some of the changes.
+For the full list please check `the release changelog <https://github.com/ethereum/solidity/releases>`_.
+
+The functions ``.call()`` (and family), ``keccak256()``, ``sha256()``, ``ripemd160()`` now accept only a single ``bytes``.
+Change every ``.call()`` to a ``.call("")`` and every ``.call(signature, a, b, c)`` to use ``.call(abi.encodeWithSignature(signature, a, b, c))`` (the last one only works for value types).
```

Comment:
Also ``call`` now returns ``(bool, bytes memory)`` - that was missing in "How to update your code" in the Changelog.